### PR TITLE
Add @types/jest, upgrade example app dependencies, README and CHANGELOG

### PR DESCRIPTION
**Pull request by AhnpGit**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,9 @@
 * Breaking: Upgrade Jest to 22 ([#109](https://github.com/thymikee/jest-preset-angular/pull/109))
 * Breaking: Upgrade `ts-jest` to 22 ([#109](https://github.com/thymikee/jest-preset-angular/pull/109))
 * Chore: Get rid of explicit `jsdom` dependency and custom test environment
+* Fix: Upgrade example app to Angular 5.2.x using Angular CLI 1.6.6
+* Fix: Add @types/jest as a package dependency
+* Fix: Adjust README for easy installation guide and add configuration section with vendor libaries like jQuery etc..
```
Comment (thymikee) [inline CHANGELOG.md]: Can you move these to master change log and link to PT?

File: README.md [hunk 2]
```diff
@@ -222,3 +224,22 @@ import 'rxjs/add/operator/catch';
 
 import './jestGlobalMocks';
 ```
+
+### Allow vendor libraries like jQuery, etc...
```
Comment (thymikee) [inline README.md]: Can we separate this into its own PR? I'd like not to mix unrelated changes in one shot

File: CHANGELOG.md (hunk 1 above)
Comment (AhnpGit) [reply CHANGELOG.md]: do you mean I should commit these changes to CHANGELOG.md on master branch and mention these changes with PR behind ?

File: README.md (hunk 2 above)
Comment (AhnpGit) [reply README.md]: ok I think that's better

File: CHANGELOG.md (hunk 1 above)
Comment (thymikee) [reply CHANGELOG.md]: No, by master I mean master changelog. The first line of this file states:
```
## Changelog (master)
```
Since these "fixes" are not part of `v5.0.0` release, let's just move them up:
```
## Changelog (master)

* Feature: Simplify installation by adding @types/jest as a package dependency ([#116](...))
* Chore: Upgrade example app to Angular 5.2 using Angular CLI 1.6 ([#116](...))

### v5.0.0
**v5.0.0 stuff, do not touch**
```
And later with a followup PR:
```
* Docs: Add a configuration section with vendor libraries like jQuery ([#117](...))
```

Comment (AhnpGit) [reply CHANGELOG.md]: ah ok I got it

**Comment by thymikee** [comment]
Thanks!
